spec/features/employee/administrator/user/administrator_delete_user_spec.rb

```diff
@@ -4,24 +4,37 @@
   so that user will be completely removed from the system
 HEREDOC
 
-RSpec.feature feature, issues: ['railgun#178'] do
+RSpec.feature feature do
```

Comment:
Original has linked issues: ['railgun#178']. This info must exist. You cannot remove it, only add more entries if needed.